src/components/App.js

```diff
@@ -67,10 +69,34 @@ class App extends Component {
 
   connectIfNeeded() {
     if (!this.props.pincodeRequired && this.props.shouldConnect) {
-      this.props.initConnection();
+      // this.props.initConnection();
```

Comment:
Can this line be removed?

Reply:
Ah, yep. Last push was some minor cleanup of three lines, including removing this.